packages/walletkit/src/contract-utils.ts

```diff
@@ -1,14 +1,20 @@
 import BigNumber from 'bignumber.js'
 import { values } from 'lodash'
+import sleep from 'sleep-promise'
+import * as util from 'util'
 import Web3 from 'web3'
 import Contract from 'web3/eth/contract'
 import { TransactionObject } from 'web3/eth/types'
 import { TransactionReceipt } from 'web3/types'
 import * as ContractList from '../contracts/index'
+import { GasPriceMinimum as GasPriceMinimumType } from '../types/GasPriceMinimum'
 import { GoldToken } from '../types/GoldToken'
 import { StableToken } from '../types/StableToken'
-import { Logger } from './logger'
+import { getGasPriceMinimumContract } from './contracts'
+import { getGoldTokenAddress } from './erc20-utils'
+import { Logger, LogLevel } from './logger'
 
+Logger.setLogLevel(LogLevel.DEBUG)
```

Comment:
Nit: Seems like this doesn't belong here

Reply:
Commented out with an explanation.